schema.json

```diff
@@ -1676,8 +1676,12 @@
 														"type": "string"
 													}
 												}
-											}
-										}
+											},
+                      "serial-group": {
+                        "type": "string",
+                        "minLength": 1
```

Comment:
~ Does it make sense to put the max length here too? Or is it unhelpful because that would apply to the source length, but the max length applies to the compiled value?

Reply:
\~~~ The surrounding code in this file appears to be using tabs for indentation, but this addition uses spaces. 

Edit: Although I notice there's not much consistency of tabs vs spaces in the other files 🤷🏼

Reply:
Yes, `maxLength` was left out for this reason, the example under implementation details is valid source and compiled config due to the variable being evaluated before we enforce those limits.